src/taskgraph/run-task/run-task

```diff
@@ -1033,6 +1033,24 @@ def _display_python_version():
         b"setup", b"Python version: %s\n" % platform.python_version().encode("utf-8")
     )
 
+def _display_subprocess_python_version():
+
+    python_versions = ["python","python2","python3"]
```

Comment:
Let's not dump `python2`, I sure hope it wouldn't be used, so I think logging about it will just add confusion.